test/Certificate.test.ts

```diff
@@ -1,97 +1,15 @@
-import type { BigNumberish, BigNumber } from 'ethers';
+// import type { BigNumberish, BigNumber } from 'ethers';
 
-import { formatTokenAmount } from '@/utils/units';
-import {
-  expect,
-  mockDepositNoriToPolygon,
-  setupTest,
-  createRemovalTokenId,
-} from '@/test/helpers';
+// import { formatTokenAmount } from '@/utils/units';
+// import {
+//   expect,
+//   mockDepositNoriToPolygon,
+//   setupTest,
+//   createRemovalTokenId,
+// } from '@/test/helpers';
 
-const setupTestLocal = async (
-  {
-    buyerInitialBPNoriBalance = formatTokenAmount(1_000_000),
-    removalDataToList = [],
-  }: {
-    buyerInitialBPNoriBalance?: BigNumberish;
-    removalDataToList?: {
-      amount: number;
-      vintage?: number;
-      supplier?: string;
-    }[];
-  } = {
-    buyerInitialBPNoriBalance: formatTokenAmount(1_000_000),
-    removalDataToList: [],
-  }
-): Promise<
-  Awaited<ReturnType<typeof setupTest>> & { listedRemovalIds: BigNumber[] }
-> => {
-  const { hre, contracts, removal, fifoMarket, ...rest } = await setupTest();
-  let tokenIds: BigNumber[] = [];
-  if (removalDataToList.length > 0) {
-    const { supplier } = hre.namedAccounts;
-    const defaultStartingVintage = 2016;
-    tokenIds = await Promise.all(
-      removalDataToList.map((removalData, index) => {
-        return createRemovalTokenId(removal, {
-          supplierAddress: removalData.supplier ?? supplier,
-          vintage: removalData.vintage ?? defaultStartingVintage + index,
-        });
-      })
-    );
-    const removalBalances = removalDataToList.map((removalData) =>
-      hre.ethers.utils.parseUnits(removalData.amount.toString())
-    );
+// describe('Certificate', () => {
+//   describe('mint', () => {
 
-    const packedData = hre.ethers.utils.defaultAbiCoder.encode(
-      ['address', 'bool'],
-      [fifoMarket.address, true]
-    );
-    await removal.mintBatch(supplier, removalBalances, tokenIds, packedData);
-  }
-  await mockDepositNoriToPolygon({
-    hre,
-    contracts,
-    amount: buyerInitialBPNoriBalance,
-    to: hre.namedAccounts.buyer,
-    signer: hre.namedSigners.buyer,
-  });
-  return {
-    hre,
-    contracts,
-    listedRemovalIds: tokenIds,
-    removal,
-    fifoMarket,
-    ...rest,
-  };
-};
-
-describe('Certificate', () => {
```

Comment:
why is this all deleted?

Reply:
just moved the test to bpNori tests since it is a function of bpNORI.send 

Was going to rewrite certificate tests as part of the scope of the certificate untransferability pr